modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensus.scala

```diff
@@ -77,7 +87,7 @@ abstract class HealthCheckConsensus[
 
       def run = onNegativeOutcome(negative) >> onNonNegativeOutcome(nonNegative)
 
-      Spawn[F].start(run)
+      run.handleErrorWith(e => logger.warn(e)("Unexpected error on outcome action"))
```

Comment:
Shouldn't this be logged as an `error` instead of a `warn`?

Reply:
We agreed that not every error should be logged as "error". This is only a fallback for the wrongly implemented `onNegative/onNonNegative` actions.

Reply:
IMO errors in implementation always should be logged on `error` level.